src/Utilities/model.jl

```diff
@@ -137,16 +145,48 @@ end
 MOI.canget(model::AbstractModel, ::MOI.Name) = true
 MOI.get(model::AbstractModel, ::MOI.Name) = model.name
 
+"""
+    newname(model::MOI.ModelLike, IndexType::Type{<:MOI.Index}, idx::MOI.Index, name::String)
+
+Return a `Bool` indicating whether `name` is not already used by an index of
```

Comment:
The negative is confusing in this description. Maybe say "Returns a `Bool` indicating whether `name` is available to be used by an index of type `IndexType` (i.e., it's not already used)."